### PR TITLE
added footer

### LINKED ISSUE #117: [Feature] add footer at bottom of the site
### Description

Hi, currently we don't have any footer in our project, and the looks incomplete without footer.

#### Specific Requirements

* should be sticky at `bottom`
* should not affect other components
* should also contain `GitHub` icon and it should point to this URL `https://github.com/rupali-codes/LinksHub/`

### Screenshots

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Aadarsh805**
## Fixes Issue
Closes #117 

## Screenshots
Screenshot from 2023-02-09 10-26-33

## Note to reviewers
Suggest changes if any

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import Link from "next/link";
+import type { FC } from "react";
+import Logo from "../logo";
+
+interface FooterProps {}
+
+export const Footer: FC<FooterProps> = ({}) => {
+  return (
+    <div className="fixed bottom-0 w-full flex items-baseline justify-center z-10 gap-1 rounded-lg bg-[#343434] py-3">
+      <p className="text-lg text-white ">
```
Comment (rupali-codes) [inline components/Footer/Footer.tsx:11]: we are not using `white` anywhere in the project. so just update it with `text-gray-200`

**Review by rupali-codes — COMMENTED** [review]
Also, can we make background `blur` instead of using this `solid` color?

**Comment by Aadarsh805** [comment]
> Also, can we make background `blur` instead of using this `solid` color?

something like this?

**Comment by rupali-codes** [comment]
Hmm.

> > Also, can we make background `blur` instead of using this `solid` color?
> 
> something like this? 

Looks good tho, but wud hv been better if cud just make it within right container. If not, just commit the changes

**Comment by Aadarsh805** [comment]
maybe what we can do is instead of keeping it fixed, I add the footer static inside the right container and make cards container have a fixed height just above to the footer
something like that
let me try

**Comment by rupali-codes** [comment]
> maybe what we can do is instead of keeping it fixed, I add the footer static inside the right container and make cards container have a fixed height just above to the footer something like that let me try

Okay

**Comment by Aadarsh805** [comment]
closes #161 as well

**Comment by Aadarsh805** [comment]
here's how it looks on mine

[screen-capture (17).webm](https://user-images.githubusercontent.com/95094057/218009377-3aad7ff5-b590-4866-bad2-460414adc1b8.webm)

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Great work Aadarsh. But having 2 small issues: 
1. It's showing 2 scrollbars 

image

2. Ig it should not be fixed this way, what i meant is when user reaches the end of the page, then we should display this footer. And for initial state (when no category is selected) it should be at the bottom of the page just like it's now.

**Comment by rupali-codes** [comment]
> closes #161 as well

And thanks for this 😊

**Comment by Aadarsh805** [comment]
> Great work Aadarsh. But having 2 small issues:
> 
> 1. It's showing 2 scrollbars
> 
> image
> 
> 2. Ig it should not be fixed this way, what i meant is when user reaches the end of the page, then we should display this footer. And for initial state (when no category is selected) it should be at the bottom of the page just like it's now.

I actually fixed the bigger scrollbar, that's exactly what is being addressed in this issue #161, I'll look on it again must be because of smaller screen size ig
and I'll try to implement it the other way as you suggested
One question, do we want the top bar fixed? i suggest it should be

**Comment by rupali-codes** [comment]
@Aadarsh805 I guess not for desktop but we can fix it t top for other devices(phone, tabs). Cz' we have sidebar in desktop so there's no need for it necessarily

**Comment by Aadarsh805** [comment]
Updated

[screen-capture (19).webm](https://user-images.githubusercontent.com/95094057/218250753-aa3b7679-a0e8-4a4c-8338-8507047da982.webm)

**Review by rupali-codes — APPROVED** [review]
Thank you so much Aadarsh, appreciate it. Now it looks perfect.

**Pull request merged**